public/controllers/management/components/management/cdblists/components/columns.tsx

```diff
@@ -0,0 +1,187 @@
+        align: 'left',
+        render: item => {
+          const defaultItems = this.props.state.defaultItems;
+          return (
+            <div>
+              <WzButtonPermissions
+                buttonType='icon'
+                permissions={getEditButtonPermissions(item)}
+                aria-label="Edit content"
+                iconType="pencil"
+                tooltip={{ position: 'top', content: `Edit ${item.filename} content` }}
+                onClick={async (ev) => {
+                  ev.stopPropagation();
+                  const resourcesHandler = new ResourcesHandler(ResourcesConstants.LISTS);
+                  const result = await resourcesHandler.getFileContent(item.filename);
+                  const file = { name: item.filename, content: result, path: item.relative_dirname };
+                  this.props.updateListContent(file);
+                }}
+                color="primary"
+              />
+              <WzButtonPermissionsModalConfirm
+                buttonType="icon"
+                permissions={getDeleteButtonPermissions(item)}
+                tooltip={{ position: 'top', content: (defaultItems.indexOf(`${item.relative_dirname}`) === -1) ? `Delete ${item.filename}` : `The ${item.filename} list cannot be deleted` }}
+                aria-label="Delete file"
+                iconType="trash"
+                isDisabled={defaultItems.indexOf(`${item.relative_dirname}`) !== -1}
+                onConfirm={async () => this.props.removeItems([item])}
+                onClose={async (ev) => ev.stopPropagation()}
+                onClick={(ev) => ev.stopPropagation()}
+                color="danger"
+                modalTitle={'Are you sure?'}
+                modalProps={{
+                  buttonColor: 'danger',
+                  onClick: (ev) => ev.stopPropagation()
+                }}
+              />
+              <WzButtonPermissions
+                buttonType='icon'
+                permissions={getReadButtonPermissions(item)}
+                aria-label="Export list"
+                iconType="exportAction"
+                tooltip={{ position: 'top', content: `Export ${item.filename} content` }}
+                onClick={async (ev) => {
+                  ev.stopPropagation();
+                  await exportCsv(`/lists`,
+                    [{ _isCDBList: true, name: 'filename', value: `${item.filename}` }],
+                    item.filename
+                  )
+                }}
+                color="primary"
+              />
+            </div>
+          )
+        }
+      }
+    };
+
+
+    this.buildColumns();
+  }
+
+  buildComplianceBadges(item) {
+    const badgeList = [];
+    const fields = ['pci_dss', 'gpg13', 'hipaa', 'gdpr', 'nist_800_53', 'tsc', 'mitre'];
+    const buildBadge = field => {
+      const idGenerator = () => {
+        return (
+          '_' +
+          Math.random()
+            .toString(36)
+            .substr(2, 9)
+        );
+      };
```

Comment:
**nitpick**: I think it is not required. We should find another way to generate the `key`.